tools/cldr-apps/js/src/esm/cldrTable.js

```diff
@@ -663,7 +663,8 @@ function extraPathAllowsNullValue(path) {
   if (
     path.includes("timeZoneNames/metazone") ||
     path.includes("timeZoneNames/zone") ||
-    path.includes("dayPeriods/dayPeriodContext")
+    path.includes("dayPeriods/dayPeriodContext") ||
+    /\/\/ldml\/units\/unitLength.*\/gender.*/.test(path)
```

Comment:
@macchiati what is the right xpath to check for here?

Reply:
`//ldml/units/unitLength[@type=\"long\"]/unit[@type=\"light-lumen\"]/gender` is my sample here